src/Bugsnag/Client.php

```diff
@@ -454,9 +454,9 @@ public function setSendCode($sendCode)
      * @param Array     $metaData  optional metaData to send with this error
      * @param String    $severity  optional severity of this error (fatal/error/warning/info)
      */
-    public function notifyException(Exception $exception, array $metaData = null, $severity = null)
+    public function notifyException($throwable, array $metaData = null, $severity = null)
```

Comment:
The documentation for this method should probably be updated to indicate a throwable is accepted here